models/stg_shopify.yml

```diff
@@ -99,7 +99,7 @@ models:
         description: The ID of the product that the line item belongs to. Can be null if the original product associated with the order is deleted at a later date.
       - name: quantity
```

Comment:
In the staging yml we'll need to update the *_timestamp fields to *_at

Reply:
good catch, added